modules/core/src/index.js

```diff
@@ -147,6 +147,7 @@ export {
   fp32,
   fp64,
   project,
+  legacyLighting,
```

Comment:
Not sure why this is part of your PR, that should already be a part of the feature branch... Maybe rebase?